omero/sysadmins/unix/install-web.txt

```diff
@@ -118,12 +118,18 @@ OMERO.web Maintenance (Unix/Linux)
           OMERO.web offers alternative session backends to automatically 
           delete stale data using the cache session store backend, see
           :djangodoc:`Django cached session documentation <topics/http/sessions/#using-cached-sessions>`
-          for more details. After installing all the cache prerequisites set the following:
+          for more details. After installing all the cache prerequisites set the following::
 
-          ::
+              $ bin/omero config set omero.web.session_engine django.contrib.sessions.backends.cache
+
+          - `Memcached <https://memcached.org/>`_::
 
               $ bin/omero config set omero.web.caches '{"default": { "BACKEND": "django.core.cache.backends.memcached.MemcachedCache", "LOCATION": "127.0.0.1:11211", "TIMEOUT": "86400" } }'
-              $ bin/omero config set omero.web.session_engine django.contrib.sessions.backends.cache
+
+          - EXPERIMENTAL: `Redis 2.4+ <http://redis.io/>`_ requires `django-redis-cache 1.6.5+ <http://django-redis-cache.readthedocs.org/en/latest/intro_quick_start.html>`_::
```

Comment:
Although the redis.io link works, the build is convinced it's broken (https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/291/warnings3Result/) - can we try it without the trailing / at the end and if Jenkins still complains, either we remove the link or I'll have to add it to the ignore list.